### ISSUE TITLE
namespace error after xacro update 

### ISSUE THREAD

**Issue by markostamos** [ethz-asl/rotors_simulator#695]
![Screenshot 2022-02-12 at 20-56-49 Facebook](https://user-images.githubusercontent.com/42519053/153724680-2b222d60-ca6e-47b0-96ab-476e28b59767.png)

there seems to be a new namespace error after the  xacro (1.14.11-1focal.20220122.020912 500) update on ros-noetic.
![Screenshot from 2022-02-12 21-02-45](https://user-images.githubusercontent.com/42519053/153724709-84a1b6ce-df50-43ea-9107-d71da29f98a7.png)

As a temp fix you can download the previous version  https://github.com/ros/xacro/archive/1.14.10.tar.gz  inside the rotors_simulator workspace/src and build it.

**Comment by CK1201** [comment]
thanks!

**Comment by johnwaneboy** [comment]
I have meet the same problem on ros-melodic. So how to solve the probelm thoroughly.

**Comment by rhaschke** [comment]
This issue has been solved upstream in xacro: https://github.com/ros/xacro/issues/309#issuecomment-1037236018
Please simply update to the latest xacro release, 1.14.13. For Melodic, a release was triggered, but not yet synced.

**Comment by danielvennestrom** [comment]
Any ETA for when the Melodic release will be synced?

**Comment by danielvennestrom** [comment]
Any estimated time for when the release for melodic will be synced?

**Comment by rhaschke** [comment]
No, @clalancette, the distro maintainer, seems to be busy and there are some unresolved regressions in Melodic blocking a new release (see https://github.com/ros/rosdistro/issues/32236).

**Comment by danielvennestrom** [comment]
I see, do you know of any workarounds for Melodic in the meanwhile?

**Comment by rhaschke** [comment]
You can pull in the latest melodic-devel or noetic-devel branch of xacro into your catkin workspace and thus use the latest xacro from source.
Alternatively, you can grab the latest released version from the testing repository: http://wiki.ros.org/TestingRepository.

**Comment by danielvennestrom** [comment]
I see, do you know of any workarounds for Melodic in the meanwhile?

**Comment by danielvennestrom** [comment]
I ended up using the testing repository to grab the latest version of xacro(http://wiki.ros.org/TestingRepository). My issue was solved!  (using ubuntu 18.04, ros-melodic).

**Comment by arnabGudu** [comment]
The new update of xacro is having an issue of not ignoring commented lines. Thus removing all commented lines is a temporary fix.
